specs/2025-02-24.yaml

```diff
@@ -1697,6 +1920,309 @@ components:
         postalCode:
           type: string
           description: "ZIP or postal code. Maximum length: 255 characters."
+    PostalAddress:
+      type: object
+      description: |-
+        A postal address is a mailing address where physical mail can be received.
+        Postal addresses can be used to receive paper checks.
+      required:
+        - city
+        - country
+        - line1
+        - postalCode
+        - state
+      properties:
+        city:
+          type: string
+          description: City, district, suburb, town, or village.
+        country:
+          type: string
+          description: Two-letter country code (https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2)
+        line_1:
+          type: string
+          description: Address line 1 (e.g. street, PO Box, or company name)
+        line_2:
+          type: string
+          description: Address line 2 (e.g. apartment, suite, unit, or building)
```

Comment:
can these be `line1` and `line2` ?

Reply:
they can, is this important?

Reply:
Also can we add `is_verified` bool to indicate wether or not the returned address is a verified payment mailing address?

Reply:
it's more cosmetic than important re: line_1 vs line1. we tend towards line1 elsewhere in the system

Reply:
Just noticed the address object on Disbursement.Transaction.DonorAdvisedFundGrant.Donors uses `line1` and `line2`. There's also inconsistencies between which address fields are optional and which are required

Reply:
okay updated